Sources/App/Controllers/Karma/KarmaParser.swift

```diff
@@ -31,13 +31,23 @@ final class KarmaParser {
         }
     }
 
-    func userIds(from message: String) -> [String] {
+    func userIdParse(from message: String) -> [String] {
         return userRegex.matches(in: message, range: NSRange(message.startIndex..<message.endIndex, in: message)).map { match in
             let groups = match.captureGroups(testedString: message)
             return groups[0]
         }
     }
 
+    func userIds(from message: String) -> [String] {
+        var ids = userIdParse(from: message)
+
+        if ids.isEmpty {
+            ids = message.split(separator: " ").map { String($0) }
```

Comment:
Can you explain the thought process behind this ? We only want to be getting the valid userIds from the message.text.

Reply:
This is a hack, which I needed because the items returned in the command text looked like this:

@jacobtrwilson @allenhumphreys 

and not like <@U12345678|jacobtrwilson> 

Since the karma command wasn't my main concern I only did enough to test that I it was not broken from the refactor itself.

Reply:
O shoot, sorry should have mentioned this since I already had to deal with this. When setting up a / command, you need to check the box to `Escape channels, users, and links sent to your app`. That will fix the issue you have and resolve the need to have this wrapper.

Reply:
updated